Rest.Services.Test/ProductRepositoryTest.cs

```diff
@@ -0,0 +1,84 @@
+using Xunit;
+using Rest.Models;
+using Rest.Services;
+using System.Collections.Generic;
+using System;
+
+namespace Rest.Services.Test;
+
+public class ProductRepositoryTest : IDisposable
+{
+    ProductRepository _sut;
+    List<Product> _products;
+
+    public ProductRepositoryTest()
+    {
+        _sut = new ProductRepository();
+        _products = new List<Product>();
+
+        for (int i = 0; i < 3; i++)
+        {
+            _products.Add(new Product(i + 1, "product" + i, i));
+        }
+    }
+
+     public void Dispose()

```

Comment:
Not sure why you need to dispose some objects in the test. Could you please elaborate?